### PR TITLE
Reduce CPU usage of data display.

### PR DESCRIPTION
**Pull request by mastykin**
Hi.
My sensor is providing many data on 100Hz,
and I see significant CPU usage compared to old CuteCom.
I think the problem is in QPlainTextEdit.
This PR is only a suggestion.
It does batch insertText() operations on timer.
CPU usage is reduced from 40% to 10%.
Dmitry

### CODE REVIEW

**Comment by mastykin** [comment]
I see that easier solution could be just:
void DataDisplay::timerEvent(QTimerEvent *event) {
		setUpdatesEnabled(true);
		setUpdatesEnabled(false);
}

But with massive output in hex mode master version (and this easier solution also doesn't help) make CueCom quickly hang when I click output text with mouse. But foregoing commits are free of this hanging.
Dmitry

File: datadisplay.cpp [hunk 1]
```diff
@@ -59,6 +59,9 @@ DataDisplay::DataDisplay(QWidget *parent)
 
     connect(m_searchPanel, &SearchPanel::findNext, this, &DataDisplay::find);
     connect(m_searchPanel, &SearchPanel::textEntered, m_highlighter, &DataHighlighter::setSearchString);
+
+    connect(&m_timer, SIGNAL(timeout()), this, SLOT(BlockReady()));
+    m_timer.start(100);
```
Comment (mastykin) [inline datadisplay.cpp]: Thank you for the comments, I will add your corrections to my branch. As I'm new to GitHub, please answer, should I do corrections as new commit, or may I make a fixup using git rebase -i and push?

**Comment by mastykin** [comment]
I agree to revert last commit (b7c5e73).  The second commit (4c56d91) is just a fixup for the first one, where I had not implemented correctly the deletion of line reminder both from m_data and from the widget.

File: datadisplay.cpp (hunk 1 above)
Comment (slapab) [inline datadisplay.cpp]: When you do rebase that means you are rewriting the history. If you will push that onto GitHub then all old commits and theirs inline comments will disappear, so please make an another commit.
